lib/index.js

```diff
@@ -78,8 +78,13 @@ internals.render = async function (template, context, options = {}) {
     const realm = (isServer ? this.realm : this.route.realm);
 
     const manager = internals.nearestManager(realm);
-
     Hoek.assert(manager, 'Missing views manager');
+
+    if (!isServer) {
+        // this is the request
+        return await manager.render(template, context, options, this);
```

Comment:
Why not pass the request every time ?

Reply:
I thought there wouldn't be a request to pass if somebody called `server.render`

Reply:
It would be good to pass it whenever available, though

Reply:
No idea, I just found it weird to only have it sometimes :)

Reply:
Oh gotcha loL -- yeah it is weird. I can recheck this to see if there are other opportunities to pass it

Reply:
@Marsup I poked around at the `server.render()` decorator func, I don't think there's a way to pass a request since it's not associated with any given request. I can pass something else though probably, but not if it's _just cuz_